src/store/metadata/stub/content/editions/apprenticeships-achievements_2023-05.json

```diff
@@ -0,0 +1,117 @@
+{
+    "@context": "https://staging.idpd.uk/#ns",
+    "@id": "https://staging.idpd.uk/datasets/apprenticeships-achievements/editions",
+    "@type": "hydra:Collection",
+    "title": "Apprenticeships achievements",
+    "editions": [
+      {
+        "@id": "https://staging.idpd.uk/datasets/apprenticeships-achievements/editions/2023-05",
+        "@type": "dcat:Dataset",
+        "in_series": "https://staging.idpd.uk/datasets/apprenticeships-achievements",
+        "identifier": "2023-05",
+        "title": "Apprenticeships achievements",
+        "summary": "Apprenticeships achieved by adults aged 16+ based on home address, rate per 100,000 population, England, academic years 2017/18 to 2022/23.",
+        "description": "This dataset shows apprenticeships achieved by adults aged 16+ based on home address, rate per 100,000 population, in England, for full academic years 2017/18 to 2022/23.",
+        "publisher": "https://staging.idpd.uk/publishers/office-for-national-statistics",
+        "creator": "https://staging.idpd.uk/publishers/department-for-education",
+        "contact_point": {
+          "name": "Apprenticeships achievements",
+          "email": "mailto:apprenticeships-achievements@ons.gov.uk"
+        },
+        "topics": [
+          "https://staging.idpd.uk/topics/children-education-and-skills"],
+        "frequency": "annual",
+        "keywords": [
+          "apprenticeship",
+          "community",
+          "learning",
+          "education",
+          "training"
+        ],
+        "licence": "http://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/",
+        "issued": "2023-05-17T12:25:28Z",
+        "modified": "2023-12-08T13:26:00Z",
+        "spatial_resolution":["E06", "E07", "E08", "E09", "E11"],
+        "spatial_coverage": "E92000001",
+        "temporal_resolution": ["P1Y"],
+        "temporal_coverage": {
+          "start": "2017-01-01T12:00:00Z",
+          "end": "2022-12-31T12:00:00Z"
+        },
+        "versions_url": "https://staging.idpd.uk/datasets/apprenticeships-achievements/editions/2023-05/versions",
+        "versions": [
+          {
+            "@id": "https://staging.idpd.uk/datasets/apprenticeships-achievements/editions/2023-05/versions/1",
+            "issued": "2023-05-17T12:25:28Z"
+          }
+        ],
+        "next_release": "2024-05-01T09:30:00Z",
+        "table_schema": {
+          "columns": [
```

Comment:
Comparing this to the dataset it appears there are some undefined columns, such as: mission, polarity, notes, etc.

Reply:
The undefined columns - mission, polarity, notes, etc are set to false on https://github.com/GSS-Cogs/family-ess/blob/main/apprenticeships-achievements/apprenticeships-achievements.json

So, how should we define and map the columns on the editions and versions file for idpd-api project?

Reply:
Great; thanks Mike